managers/signals.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from typing import Any
+from typing import Set
+
+from django.db.models.signals import m2m_changed
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from employees.models import TaskActivityType
+from managers.models import Project
+from users.models import CustomUser
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(m2m_changed, sender=Project.managers.through)
+def update_user_type(sender: Project, action: str, pk_set: Set, **kwargs: Any) -> None:
```

Comment:
Those are not exactly signals, but signals handlers. What was the case for them being in models, like it is usually done?

Reply:
@rwrzesien Obsługiwały sytuację, gdy ktoś tworzył projekt i nadawał użytkownikowi `user_type=Manager` lub zabierał prawa jako menedzer jezeli nie mial innych projektów.

Reply:
I use this name to file because It was explained like that here:
https://simpleisbetterthancomplex.com/tutorial/2016/07/28/how-to-create-django-signals.html

Reply:
I get the point, but personally I think placing them under the models works fine. I have never experienced any import issues like they mention in this article. I think it is readable when you have signals right under the model definition, as it is actually a part of model "logic".
But if you decide to move them away to separate file it is OK for me too. But honestly in `signals.py` I would rather expect custom signals objects (https://docs.djangoproject.com/en/2.2/topics/signals/#defining-signals).

Reply:
Sure I will make research about the problem with this import but for one I don't know how I could get round this problem

Reply:
I think it is resolved now?